include/arch/arm/aarch64/timer.h

```diff
@@ -45,6 +46,22 @@ static ALWAYS_INLINE void arm_arch_timer_enable(unsigned char enable)
 			 : : "r" (cntv_ctl) : "memory");
 }
 
+static ALWAYS_INLINE void arm_arch_timer_irq_mask(unsigned char mask)
```

Comment:
It is often desirable to name the functions that set/get values accordingly.

```suggestion
static ALWAYS_INLINE void arm_arch_timer_set_irq_mask(unsigned char mask)
```